app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
+      # Log the user in and redirect to the user"s show page.
+      log_in user
+      redirect_to user
+    else
+      # Create an error message.
+      flash.now[:danger] = "Invalid email/password combination"
+      render "new"
```

Comment:
dùng symbol nhé, so sánh sự khác nhau giữa việc dùng symbol và string
```suggestion
      render :new
```